cunumeric/config.py

```diff
@@ -86,6 +86,7 @@ class CuNumericOpCode(IntEnum):
     BINCOUNT = _cunumeric.CUNUMERIC_BINCOUNT
     CONVERT = _cunumeric.CUNUMERIC_CONVERT
     CONVOLVE = _cunumeric.CUNUMERIC_CONVOLVE
+    CHOOSE = _cunumeric.CUNUMERIC_CHOOSE
```

Comment:
Please move this according to alphabetical order (above CONVERT)

Reply:
Will do